### PR TITLE
Altair support

### PR DESCRIPTION
**Pull request by ferranbt**
This PR adds support for the new types in Altair as per version v1.1.0 of the spectests.

New types:

- [x] Byte slices not multiple of 32 bytes (i.e SyncCommittee)
- [x] Bitlist with fixed size (i.e. SyncAggregate)
- ...

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Any update on this? we are hoping to unblock our atlair code for final review as soon as possible.

Let us know how we can help

**Comment by prestonvanloon** [comment]
Any update on this? We're hoping to close out remaining Altair items this week.

**Comment by kasey** [comment]
@ferranbt github didn't create a backlink from this PR comment, so I'll link it here for visibility https://github.com/ferranbt/fastssz/pull/60#issuecomment-908540639

File: sszgen/hash.go [hunk 1]
```diff
@@ -45,11 +45,17 @@ func (v *Value) hashRoots(isList bool, elem Type) string {
 	var elemSize uint64
 	if elem == TypeBytes {
 		// [][]byte
-		appendFn = "Append"
-		elemSize = 32
+		if v.e.s != 32 {
+			// we need to use PutBytes in order to hash the result since
+			// is higher than 32 bytes
+			appendFn = "PutBytes"
```
Comment (kasey) [inline sszgen/hash.go:51]: Should this be `v.e.s % 32 != 0`?

**Review by kasey — CHANGES_REQUESTED** [review]

**Pull request merged**